src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve();
```

Comment:
```suggestion
    resolve('Promise was resolved!');
```
in a similar way as you handle error